src/modalities/tokenization/tokenizer_wrapper.py

```diff
@@ -189,8 +218,8 @@ def tokenize(self, text: str) -> list[int]:
         Returns:
             list[int]: List of token IDs.
         """
-        tokens = self.tokenizer.encode(text)
-        return tokens
+        token_ids = self.tokenizer.Encode(text)
```

Comment:
Why are all the SP tokenizer methods written with an uppercase first letter? It seems to me that the standard is lowercase: https://github.com/google/sentencepiece/tree/master/python#segmentation